server/scsynth/SC_SequencedCommand.cpp

```diff
@@ -1146,7 +1146,7 @@ bool BufCloseCmd::Stage2()
 {
 #ifdef NO_LIBSNDFILE
 	SendFailure(&mReplyAddress, "/b_close", "scsynth compiled without libsndfile\n");
- 	scprintf("scsynth compiled without libsndfile\n");
+	scprintf("scsynth compiled without libsndfile\n");
```

Comment:
Side note--I smell a possible refactor here, especially with the repeated `#ifdef`s.